src/mscorlib/src/System/Collections/Concurrent/ConcurrentQueue.cs

```diff
@@ -1109,12 +1109,23 @@ internal struct Slot
         }
     }
 
+    /// <summary>A placeholder class for common padding constants and eventually routines.</summary>
+    internal static class PaddedHelper
```

Comment:
Environment is a dumping ground for everything. It may be better to keep it as a small type with a constant. It does not really matter because of it has to be internal in the current shape, and thus we can change it anytime.